src/js/components/Accordion/stories/typescript/CustomHeader.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import isChromatic from 'storybook-chromatic/isChromatic';
+
+import {
+  Accordion,
+  AccordionPanel,
+  Box,
+  Grommet,
+  Text,
+  TextInput,
+} from 'grommet';
+import { grommet } from 'grommet/themes';
+
+const renderPanelHeader = (title, active) => (
+  <Box direction="row" align="center" pad="medium" gap="small">
+    <strong>
+      <Text>{title}</Text>
+    </strong>
+    <Text color="brand">{active ? '-' : '+'}</Text>
+  </Box>
+);
+
+const CustomHeaderAccordion = () => {
+  const [activeIndex, setActiveIndex] = useState([0]);
+
+  return (
+    <Grommet theme={grommet}>
+      <Accordion
+        activeIndex={activeIndex}
+        onActive={newActiveIndex => setActiveIndex(newActiveIndex)}
```

Comment:
```suggestion
        onActive={(newActiveIndex: number) => setActiveIndex(newActiveIndex)}
```
should we be more typescript oriented and add the type?

Reply:
Hey Shimi, my inclination here is that we don't need to redefine the type because this type has already been defined in the index.d.ts, so in the code we wouldn't have to set it again. Because it's inline, if newActiveIndex was coming in as a string, for example, it would flag it. It seems like this is similar to this example on the typescript docs:
<img width="882" alt="Screen Shot 2019-11-01 at 10 26 14 AM" src="https://user-images.githubusercontent.com/12522275/68043354-56036a80-fc92-11e9-8975-7b826bd68de8.png">
They define the types in their interface, then in the code they don't redefine them. However, in the case that the handler was defined in a separate line of code, then the editor will flag you to specify the type. For example, if we have it separate like this then it will ask us to specify the type:
```
const onActive = (newActiveIndex: number[]) => { setActiveIndex(newActiveIndex) }

<Accordion onActive={onActive}>
...
```
When it is inline, though, there is no flag, so it seems like it is able to infer the type from the index.d.ts file.